server.js

```diff
@@ -1,17 +1,43 @@
+require('newrelic');
 const express = require('express');
 const morgan = require('morgan');
 const path = require('path');
 const cors = require('cors');
 
+const React = require('react')
+
+const ReactDOMServer = require ('react-dom/server');
+
+const Page  = require('./htmlTemplates/page.js')
+const Body =  require('./htmlTemplates/body.js')
+const Scripts =  require('./htmlTemplates/scripts.js')
+const rawComponents = require('./bundles.js')
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 3008;
 
 
 app.use(morgan('dev'));
 app.use(cors());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use('/item/lib', express.static(path.join(__dirname, 'public/lib')));
+
+app.get('/item/:id', (req, res) => {
+
+  const components = Object.entries(rawComponents).map(component=>{
+    return {
+      name: component[0],
+      react: React.createElement(component[1], {projectId: req.params.id})
+    }
+  })
```

Comment:
Several things here:
1. Destructuring each component to [name, class] or similar would make this cleaner. 
2. param => { return { foo: bar} } is redundant, just use param => ({ foo: bar }).
3. 'react' is a confusing property name for the created element.